channel-email/src/test/java/com/synopsys/integration/alert/channel/email/attachment/compatibility/MessageContentGroupTest.java

```diff
@@ -0,0 +1,137 @@
+package com.synopsys.integration.alert.channel.email.attachment.compatibility;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.LinkedHashSet;
+import java.util.Optional;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.synopsys.integration.alert.common.enumeration.ItemOperation;
+import com.synopsys.integration.alert.common.message.model.LinkableItem;
+
+public class MessageContentGroupTest {
+    public static final String PROVIDER_NAME = "test-providerName";
```

Comment:
we can also drop the public from any of the static final fields as they are only used in tests and aren't accessed by another class

Reply:
fixed